.flake8

```diff
@@ -0,0 +1,17 @@
+[flake8]
+max-line-length = 120
```

Comment:
I saw [this](https://github.com/kserve/kserve/blob/96ca17a152d4603b1c2b52aa5bd02ff6d2f45140/python/kserve/setup.cfg#L2), is it something still required? maybe can delete it.

Reply:
Yep. Deleted.